### ISSUE TITLE
Better docs

### ISSUE THREAD

**Issue by stephentcannon** [thespacedojo/simple-form#7]
Needs better docs in the README.md  

People are not always online while coding and cannot read online docs.

Local reference is important.

**Comment by queso** [comment]
I agree, I actually would like to get something like TomDoc going on.  I am a fan of Backbone doc style (using code comments), perhaps I should look at what processor they are using.

**Comment by dandv** [comment]
:+1: (typing this from a crappy [GoGoInflight connection](http://outkastz.com/gogo-in-flight-wireless-internet-free/))

**Comment by ryw** [comment]
real issue here is that our docs aren't in our README — @queso didn't we have a solution to both have better README + still have our pretty github.differential.io pages too, from the same source?

**Comment by queso** [comment]
Yeah, we found one, but it would negatively impact SEO.  I found a new solution in the form of an npm binary that would do some git-fu to sync things up, should find some time to do it.

## Either way @danv is right, we need better docs for sure, the README still isn’t enough anymore.

Josh Owens |  Partner at Differential | 513-678-7081 | @joshowens

On June 27, 2014 at 7:43:59 PM, Ry Walker (notifications@github.com) wrote:

real issue here is that our docs aren't in our README — @queso didn't we have a solution to both have better README + still have our pretty github.differential.io pages too, from the same source?

**Comment by iambrian7** [comment]
The documentation link has 404. How do I separate "label" from field?

OK - I figured out the label field, {{text_field ' name' label= 'Name of Meeting'}}
